src/index.html

```diff
@@ -6,17 +6,69 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>MOYO header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="./style.css"
-    />
+    >
```

Comment:
The `link` tag attributes are not formatted correctly. According to the checklist, if an HTML element has long attribute values or a number of attributes is more than 2, you should start each one, including the first, on a new line with a 2-space indentation related to the tag. The tag’s closing bracket should be on the same level as the opening one.